docs/overrides/hooks/anchors.py

```diff
@@ -36,5 +36,5 @@ def override_include_markdown_link(html, page, config, files):
 
 def on_page_content(html, page, config, files):
     html = override_absolute_path(html, page, config, files)
-    html = override_include_markdown_link(html, page, config, files)
+    # html = override_include_markdown_link(html, page, config, files)
```

Comment:
is this comment necessary?

Reply:
I was commenting it out for testing. Fixed it.